building/tracks/syllabus/README.md

```diff
@@ -0,0 +1,177 @@
+# Syllabus
+
+A fully featured Exercism track has two types of exercises: Practice Exercises and Concept Exercises.
+Concept Exercises and Practice Exercises are fundamentally different and complement each other well.
+
+A track's [Concept Exercises](/docs/building/tracks/concept-exercises) are exercises designed to teach specific concepts that form the basis of a programming language.
+These concepts form a _syllabus_.
+
+This documentation contains pointers and tips on how to succesfully design a syllabus for your track.
+
+## The goal of a syllabus
+
+The end goal of a syllabus is to lead students to be comfortable with idiomatic code in the target language.
+
+Each individual Concept Exercise is very tightly focused.
+It is a very small step that moves the student towards understanding something about the language.
+It builds only on concepts that have been introduced previously.
+
+By solving the exercise, the student begins the process of becoming familiar with the concept.
+Understanding comes primarily through doing, much less so through explanations.
+The explanations are only there so that the student can do what needs to be done.
+
+We want to allow students to start writing code immediately, without having to understand everything up front.
+In order to achieve this we hand-wave over details and we leave a lot of things unexplained.
+We simplify and provide code stubs where possible.
+This reduces the cognitive burden of getting started and provides the time and space for the knowledge to sink in.
+By taking this approach we're not saying that the student doesn't need to know these things, we're saying that they don't need to know them _yet_.
```

Comment:
```suggestion
We want to allow students to start writing code immediately, without having to understand everything up front.
In order to achieve this we hand-wave over details and we leave a lot of things unexplained.
We simplify and provide code stubs where possible.
This reduces the cognitive burden of getting started and provides the time and space for the knowledge to sink in.
By taking this approach we're not saying that the student doesn't need to know these things, we're saying that they don't need to know them **yet**.
```

`**` renders much more firmly than `_` on the website (bold rather than italic) - so I prefer it for making strong points like this.